.github/workflows/ci.yml

```
@@ -1,128 +1,342 @@
 name: Omega_KG CI Pipeline
 
 on:
+  push:
+    branches: [ "beta", "main" ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)